### DISCUSSION TITLE
1.18 Release

### DISCUSSION THREAD

**Discussion by Draylar** [Draylar/wolves-with-armor#40] [Announcements]
Wolves with Armor has been released for 1.18. If you have any questions, feel free to leave them here!

**Comment by tamrind69** [comment]
Hi! thanks a lot for the mod . Could you pls specify what other features the mod brings as I couldn't find any documentation.
Thanks in advance.

**Reply by Draylar** [reply]
The mod only provides Wolf Armor (Leather, Iron, Gold, Diamond, Netherite). You can add protection to it through an anvil. :)

**Reply by lxrdjuni** [reply]
hey I wanted to use your mod for fabric, but for some reason I can't equip the wolves with any of the armors. I tried going in creative and survival, and even crafting the armor myself. I right click them holding the armor and nothing happens is there something I'm doing wrong or is this a bug?

**Reply by Draylar** [reply]
If you are playing on a server, do you have the mod installed on your client _and_ server? It would also be helpful if you could send your game logs for me to check out - if you're using the vanilla launcher, you can follow [these instructions](https://minecrafthopper.net/help/guides/getting-minecraft-game-output-log/), or just copy from the output log if you are using MultiMC. Make sure the Wolf you are right-clicking has been tamed by you (it cannot be a wild Wolf or another player's Wolf).

**Comment by mkehoe28** [comment]
Love your mod and appreciate the work you put into this. Any chance  of an update to 1.18.2?

**Comment by Saturnlock** [comment]
How do I actually use the mod. I added it to my datapacks folder on a world but it's not working. Is there a command that I need to use in order for it to actually turn on?

**Comment by ghost** [comment]
Hello, I downloaded the mod and when I go to play, the world does not create/load, what can it be?
